Library/Formula/imagemagick.rb

```diff
@@ -1,13 +1,12 @@
 class Imagemagick < Formula
   desc "Tools and libraries to manipulate images in many formats"
-  homepage "http://www.imagemagick.org"
+  homepage "https://www.imagemagick.org/"
   # Please always keep the Homebrew mirror as the primary URL as the
   # ImageMagick site removes tarballs regularly which means we get issues
   # unnecessarily and older versions of the formula are broken.
-  url "https://dl.bintray.com/homebrew/mirror/ImageMagick-6.9.3-0.tar.xz"
-  mirror "http://www.imagemagick.org/download/releases/ImageMagick-6.9.3-0.tar.xz"
-  sha256 "2ba0656eb03d72d8a44e741ead524e8c34097418c0bb5487a5c4f4fe5eca9656"
-  revision 2
+  url "https://dl.bintray.com/homebrew/mirror/ImageMagick-6.9.3-5.tar.xz"
+  mirror "https://www.imagemagick.org/download/releases/ImageMagick-6.9.3-5.tar.xz"
```

Comment:
Not sure if you misread the version or if upstream did a re-release, but we're now up to:

```
https://www.imagemagick.org/download/ImageMagick-6.9.3-6.tar.xz
```
As the latest upstream.